ontopy/ontology.py

```diff
@@ -1270,6 +1273,60 @@ def addancestors(entity, counter, subject):
             return ancestors.difference(classes)
         return ancestors
 
+    def get_descendants(
+        self,
+        classes: "Union[List, ThingClass]",
+        common: bool = False,
+        generations: int = -1,
+        all_descendants: bool = False,
```

Comment:
Personally I find a single option with a fill value simpler than two closely coupled options. 

I would therefore have removed `all_descendants` and made `generations` equal -1 or (better) None mean to include all generations in the output. Since there are no obvious reason for a particular default value for `generations`, I would have made returning all descendants the default.

However, all of this is of cause a matter of taste, which you can take or leave as you wish.

Reply:
I was thinking to have it different from entity.descendants as we  initally wanted this to get the first generation. But I agree with you that it is better to have all as default.